art-common/README.md

```diff
@@ -0,0 +1 @@
+# commonlib - ART common library
```

Comment:
What is the purpose of this directory? Think the Makefile can or should be at the project root, and then there is no reason for this directory?

Reply:
added a Makefile in the top level, but for now keeping this as a separate project.